### PR TITLE
adding support for proxies env

### PR DESCRIPTION
**Pull request by ravid1323**
added the option to use proxies address if needed.
the proxies has to be in form of dictionary like the following:

{
'http' :  'address',
'https' :  'address'
}

### CODE REVIEW

**Comment by rhymiz** [comment]
Thanks for you contribution @ravid1323 !

We'll get this reviewed ASAP - in the mean time, could you also update the README (in this PR) to include the option to pass along proxy configuration?

**Comment by ravid1323** [comment]
@rhymiz Hi, I upladed an update for the README file

**Review by chdastolfo — COMMENTED** [review]
I think in general I'm wondering whether this belongs in this library. While I certainly see the utility, wanting to using a proxy with requests is not functionality specific to monday.com. I think this is an instance where forking the repo could make sense, unless there were a lot of people requesting the feature/there was evidence that it was a common enough positive use case. I'm also curious whether other, similar libraries support this.

@rhymiz curious what you think?

File: README.md [hunk 1]
```diff
@@ -134,7 +148,9 @@ monday.items.create_item(board_id='12345678', group_id='today',  item_name='Do a
     <tr>
       <td align="center"><a href="https://github.com/pevner-p2"><img src="https://avatars.githubusercontent.com/u/45570949?v=4?s=100" width="100px;" alt="pevner-p2"/><br /><sub><b>pevner-p2</b></sub></a><br /><a href="https://github.com/ProdPerfect/monday/commits?author=pevner-p2" title="Code">💻</a></td>
       <td align="center"><a href="https://github.com/t-a-y-l-o-r"><img src="https://avatars.githubusercontent.com/u/32030464?v=4?s=100" width="100px;" alt="Taylor Cochran"/><br /><sub><b>Taylor Cochran</b></sub></a><br /><a href="https://github.com/ProdPerfect/monday/commits?author=t-a-y-l-o-r" title="Code">💻</a></td>
-    </tr>
+      <td align="center"><a href="https://github.com/ravid1323"><img src="https://avatars.githubusercontent.com/u/4204232?s=400&v=4?s=100" width="100px;" alt="ravid1323"/><br /><sub><b>Ravid Maduni</b></sub></a><br /><a href="https://github.com/ProdPerfect/monday/commits?author=ravid1323" title="Code">💻</a></td>
```
Comment (chdastolfo) [inline README.md:151]: For consistency we should just let our bot add this, no?

**Comment by rhymiz** [comment]
@chdastolfo I think we're in agreement here - there are alternative ways, which I proposed in this issue https://github.com/ProdPerfect/monday/issues/75. 
Every one-off feature becomes something we will need to support indefinitely (or until we decide to deprecate it)

**Comment by chdastolfo** [comment]
Ah my bad @rhymiz, didn't see your comment on the issue. Since we're on the same page, I'm going to close this (and the corresponding issue) out. 

cc @ravid1323 hopefully our reasoning is clear here.

**Comment by ravid1323** [comment]
Well the main reason to use this feature is for company uses. Most of the big companies using proxy in their labs for security reasons.
I know that there are already some that didn't use the client for that reason, and used request with the query for doing so.
I think for anyone else it's a nice to have, and won't actually do any harm to merge it, but oh well. Thanks anyway for the opportunity

**Pull request closed without merging**